monasca/values.yaml

```diff
@@ -498,11 +524,7 @@ alarms:
     retries: 24
     delay: 5
   keystone:
-    username: mini-mon
```

Comment:
I'd recommend keeping the verbose options here and dropping `defaults_enabled` - that's more a workaround for monasca-docker than for helm.